pkg/render/machine-api-operator-config.yaml

```diff
@@ -1,6 +1,12 @@
 apiVersion: v1
```

Comment:
We should move this into the examples folder

Reply:
Done.